src/main/java/com/example/demo/model/Book.java

```diff
@@ -0,0 +1,22 @@
+package com.example.demo.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data
```

Comment:
don't use `@Data` for entities
Add `@Table`

```suggestion
@Getter
@Setter
```